### ISSUE TITLE
youtube-dl options is incomplete

### ISSUE THREAD

**Issue by jeffreykennethli** [mkly/youtube-closed-captions#12]
Currently, the youtube-dl options that ytcc downloads with are:

```
self.opts = {
        'skip_download': True,
        'writeautomaticsub': True,
        'outtmpl': 'subtitle_%(id)s',
        'logger': FakeLogger()
}
```
However, there are many more options for youtube-dl captions that should be considered, see https://github.com/ytdl-org/youtube-dl/blob/6423d7054eb2a73a1557c8531f631330e6d7e084/youtube_dl/options.py#L426-L437.

The major downside with the current `self.opts` is if a video doesn't have automatic subs, but does have other subs, ytcc will not download it, and users will be shown a `No such file or directory: 'subtitle_<video_id>.<language>.vtt'` error (#7 and #8), such as for a video like https://www.youtube.com/watch?v=BaW_jenozKc.

Basically:
1. The error message should be clearer.
2. The API should clarify which subtitles are being downloaded, perhaps specified through an Options parameter to `get_captions`.

**Comment by mkly** [comment]
Fixed in master: updated default download options to include writesubtitles and subtitlesformat, and added DownloadException handling for missing subtitles.
